### PR TITLE
Weaponsound fix  (plays at feet)

### PR DESCRIPTION
**Pull request by SciathGH**

### CODE REVIEW

**Review by Peter-Crawley — COMMENTED** [review]
Why?

**Comment by SciathGH** [comment]
has 0 range rn

**Comment by SciathGH** [comment]
Literally cannot hear it
at all

File: src/main/kotlin/net/starlegacy/feature/starship/subsystem/weapon/projectile/SimpleProjectile.kt [hunk 1]
```diff
@@ -54,6 +54,11 @@ abstract class SimpleProjectile(
 	}
 
 	protected fun playCustomSound(loc: Location, soundName: String, chunkRange: Int, pitch: Float = 1f) {
+		loc.world.players.forEach {
+			if (it.location.distance(loc) > range){
+				loc.world.playSound(it.location, soundName, SoundCategory.PLAYERS, 1.0f, pitch)
+			}
+		}
 		loc.world.playSound(loc, soundName, SoundCategory.PLAYERS, 1.0f, pitch)
```
Comment (Peter-Crawley) [inline src/main/kotlin/net/starlegacy/feature/starship/subsystem/weapon/projectile/SimpleProjectile.kt:62]: This seems like it may play the sound twice

**Review by Peter-Crawley — CHANGES_REQUESTED** [review]

**Pull request merged**